poppy/poppy_core.py

```diff
@@ -437,16 +432,16 @@ def display(self,what='intensity', nrows=1,row=1,showpadding=False,imagecrop=Non
                 title = "Intensity "+self.location
                 title = title.replace('after', 'after\n')
                 title = title.replace('before', 'before\n')
-            plt.title(title)
-            plt.xlabel(unit)
+            ax.set_title(title)
+            ax.set_xlabel(unit)
             if colorbar: plt.colorbar(ax.images[0], orientation='vertical', shrink=0.8)
 
-            if self.planetype ==_IMAGE:
+            if is_angular:
```

Comment:
I was trying to avoid subclassing `display` since it's a substantial hunk of code. But more fundamentally the `FresnelWavefront.angular_coordinates` attribute is *not* just used for display. Its main purpose is actually to select what coordinate system is used when computing the phasor; the display stuff is secondary.

In detail: If you do e.g. `somefresnelwave *= CircularOcculter()` then `CircularOcculter.getPhasor` expects that `somefresnelwave.coordinates()` returns coordinates in arcseconds.  The `angular_coordinates` attribute is what tells it to do so. 

In some sense you can think of `angular_coordinates` as an "act as if you were an image type wavefront" toggle. 

Under the Fresnel formalism, there is no real physical distinction between image and pupil planes; everything is just a plane somewhere in space with coordinates with meters. So even if it's convenient to think of them as "oh, this one is supposed to be an image", I decided it was best to be explicit that there is a coordinate toggle happening, rather than trying to do it always implicitly only based on inference from plane types. It still does that inference as best as possible, it just does so by setting an explicit and user-visible boolean.